### PR TITLE
Initial conversion of wiki to Sphinx/RTD

### PR DESCRIPTION
**Pull request by clenk**

### CODE REVIEW

File: docs/Plugins-evals.md [hunk 1]
```diff
@@ -0,0 +1,241 @@
+Plugin: evals
+==========
+
+## APT3 DIY ATT&CK Evaluation Round 1
+
+ATT&CK Evaluations use an Operational Flow to describe the logical ordering to which procedures take place during the evaluation. Round 1's Operational Flow is found [here](https://attackEval.mitre.org/methodology/round1/operational-flow.html).
```
Comment (clenk) [inline docs/Plugins-evals.md]: This link is broken and I'm not sure what the updated link should be.

File: docs/Plugins-evals.md [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+The `reg` utility is executed via `cmd` to enumerate a specific Registry key associated with local system policies to ensure that the user will not be prompted for credentials when elevating permissions.
+
+#### Step 3 - Privilege Escalation
+
+Step 3 tested the following techniques:
+
+##### 3.A.1 - Bypass User Account Control (T1088) via Access Token Manipulation (T1134)
+This Eval plugin ability uses [Empire's Invoke-BypassUACTokenManipulation.ps1](https://github.com/EmpireProject/Empire/blob/master/data/module_source/privesc/Invoke-BypassUACTokenManipulation.ps1) script to steal the token of an existing high-integrity process and launch a new, high-integrity CALDERA RAT with limited functionality.
+
+\*Note if a high integrity process is not running the script will attempt to spawn TaskMgr.exe with the RunAs flag. If there is no high integrity process running or TaskMgr.exe fails to spawn as admin the script will fail. We recommend spawning cmd.exe as the high integrity process prior to starting the DIY Eval. Read more about the lab setup [here]()
```
Comment (clenk) [inline docs/Plugins-evals.md]: Not sure where this link was meant to point to.

File: docs/Plugins-evals.md [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+
+Step 3 tested the following techniques:
+
+##### 3.B.1 - Process Discovery (T1057) and 3.C.1 - Process Injection (T1055)
+The limited functionality high-integrity RAT injects malicious base64 encoded PowerShell into an existing fully functional high-integrity process, resulting in a new elevated, fully functional high-integrity RAT.
+
+This Eval plugin ability uses [Empire's Invoke-PSInject](https://github.com/EmpireProject/PSInject/blob/master/Invoke-PSInject.ps1) to perform the injection. This  ability added the [EnumProcesses function from Empire's Invoke-BypassUACTokenManipulation.ps1](https://github.com/EmpireProject/Empire/blob/master/data/module_source/privesc/Invoke-BypassUACTokenManipulation.ps1#L876) script in, as it now allows CALDERA to find a high integrity process to inject in to on the fly.
+
+\*Note like Step 3.A if there is no high integrity process running and owned by the current user the script will fail. We recommend spawning cmd.exe as the high integrity process prior to starting the DIY Eval. Read more about the lab setup [here]()
```
Comment (clenk) [inline docs/Plugins-evals.md]: Not sure where this link was meant to point to.

File: docs/Plugins-evals.md (hunk 1 above)
Comment (wbooth) [reply docs/Plugins-evals.md]: https://attackevals.mitre.org/methodology/round1/operational-flow.html

File: docs/Plugins-evals.md (hunk 2 above)
Comment (wbooth) [reply docs/Plugins-evals.md]: https://attackevals.mitre.org/methodology/round1/environment.html

File: docs/Plugins-evals.md (hunk 3 above)
Comment (wbooth) [reply docs/Plugins-evals.md]: https://attackevals.mitre.org/methodology/round1/environment.html

**Pull request merged**